weedcoco/importers/mask.py

```diff
@@ -127,11 +113,64 @@ def _image_name_to_mask(name):
                     f"No mask found at {mask_path} for image named {path.name}."
                 )
             continue
+
+        yield mask_path, path
+
+    if not all_paths:
+        raise ValueError(f"No images found in {image_dir}")
+
+
+def generate_paths_from_mask_only(mask_dir: Path, image_ext: str = "jpg"):
+    all_paths = sorted(mask_dir.glob("*.*"))
+    for mask_path in all_paths:
+        if mask_path.name.startswith("."):
+            # real glob excludes these
+            continue
+        image_file_name = f"{mask_path.stem}.{image_ext}"
+        yield mask_path, image_file_name
+
+    if not all_paths:
+        raise ValueError(f"No masks found in {mask_dir}")
+
+
+def masks_to_coco(
+    path_pairs: Iterable[Tuple[str, str]],
+    color_to_category_map: Mapping[str, str],
+    check_consistent_dimensions: bool = True,
+):
+    """Converts images and masks to MS COCO images and annotations
+
+    Parameters
+    ----------
+    path_pairs : Iterable
```

Comment:
In this case the api of this module will be main function? or it's not the final version. I was expecting the arguments with only mask_dir as required and image_ext as optional

Reply:
You're welcome to use it as a command-line tool, or by using this function and passing in the iterable from `generate_paths_from_mask_only`